integrations/standalone/tests/pageobjects/Table.ts

```diff
@@ -0,0 +1,90 @@
+import { Locator, Page, expect } from '@playwright/test';
+
+export class Table {
+  private readonly label: string | undefined;
+  private readonly parentLocator: Locator;
+  private readonly page: Page;
+  private readonly rows: Locator;
+
+  constructor(page: Page, parentLocator: Locator, label: string | undefined) {
+    this.page = page;
+    this.parentLocator = parentLocator;
+    this.label = label;
+    this.rows = this.tableLocator().locator('tbody').getByRole('row');
+  }
+
+  async addRow() {
+    await this.tableLocator().getByRole('row', { name: 'Add row' }).click();
+    return this.row(0);

```

Comment:
This will only work if there was no row before. Maybe there is already one, so we should return the latest row here.